AppCenter/AppCenter/Internals/Ingestion/MSOneCollectorIngestion.m

```diff
@@ -94,6 +97,40 @@ - (NSURLRequest *)createRequest:(NSObject *)data
                     stringWithFormat:@"%lld",
                                      (long long)[MSUtility nowInMilliseconds]]
          forKey:kMSOneCollectorUploadTimeKey];
+
+  // Gather tokens from logs.
+  NSMutableString *ticketKeyString = [NSMutableString new];
+  for (id<MSLog> log in container.logs) {
+    MSCommonSchemaLog *csLog = (MSCommonSchemaLog *)log;
+    if (csLog.ext && csLog.ext.protocolExt) {
+      NSArray<NSString *> *ticketKeys = [[[csLog ext] protocolExt] ticketKeys];
+      if (ticketKeys) {
+        for (NSString *ticketKey in ticketKeys) {
+          NSString *token =
```

Comment:
nit - unsplit this line

Reply:
The format seems correct here for the LLVM default code formatter.